app/uk/gov/hmrc/helptosavefrontend/config/FrontendAppConfig.scala

```diff
@@ -99,4 +99,14 @@ class FrontendAppConfig @Inject() (override val runModeConfiguration: Configurat
   val reportAProblemNonJSUrl: String = s"$contactBaseUrl/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
   val betaFeedbackUrlNoAuth: String = s"$contactBaseUrl/contact/beta-feedback-unauthenticated?service=$contactFormServiceIdentifier"
   val govUkExitURL: String = getString("gov-uk.url")
+
+  object BankDetailsConfig {
+    val sortCodeLength: Int = getInt("bank-details-validation.sort-code.length")
```

Comment:
also isn't `sort-code` just word , like `sortCode` and so hyphen is not required for it , but this needs to change at lot of places, so we can live with it for now 👍